stats/cloud/client.go

```diff
@@ -125,14 +127,17 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 	return err
 }
 
-func (c *Client) do(req *http.Request, v interface{}, attempt int) (retry bool, err error) {
+func (c *Client) prepareHeader(req *http.Request) {
```

Comment:
Since we're dealing with multiple headers, can you call this `prepareHeaders`?

Reply:
Fixed